src/libraries/System.Text.Json/src/System/Text/Json/Serialization/DefaultReferenceResolver.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the.NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace System.Text.Json
+{
+    internal sealed class DefaultReferenceResolver
+    {
+        private uint _referenceCount;
```

Comment:
Is `_referenceCount` used anywhere? I see a ++ done on it, but not using it otherwise.